bin/fin

```diff
@@ -5954,7 +5974,18 @@ alias_list ()
 load_global_configuration ()
 {
 	export DOCROOT=${DOCROOT:-docroot}
+	export DOCKSAL_PATH="$(get_project_path)"
 	export PROJECT_ROOT="$(get_project_path)"
+	is_windows && export PROJECT_ROOT_WIN="$(get_project_path_dc)"
+
+	# On macOS 10.15 Catalina special treatment
+	# See https://github.com/docksal/docksal/issues/1130
+	# The exported NFS path needs the "/System/Volumes/Data" prefix unless project is on an external drive (/Volumes/...)
+	if is_mac_catalina && ! [[ "${PROJECT_ROOT}" =~ "/Volumes" ]]; then
```

Comment:
So the only firmlink we need to handle here is actually `/Users`, so I guess we should specifically target it instead of targeting 'not Volumes'. 
Also we need to check beginning of the string specifically in regexp

```suggestion
	if is_mac_catalina && [[ "${PROJECT_ROOT}" =~ ^/Users ]]; then
```

Reply:
Actually in the light of what discussed in DM we don't need this at all. Instead of this we need to modify `upfind` so that on Catalina it returned `/System/Volumes/Data/Users` instead of `/Users`